### PR TITLE
bump golangci-lint to v1.50.1

### PR DESCRIPTION
**Pull request by vrothberg**
Also fix a number of duplicate words.  Yet disable the new `dupword` linter as it displays too many false positives.

Signed-off-by: Valentin Rothberg <vrothberg@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ linters:
     # All these break for one reason or another
     - tagliatelle # too many JSON keys cannot be changed due to compat
     - nosnakecase # too many false positives due to the `unix` package
+    - dupword # too many false positives (e.g., in tests)
```
Comment (Luap99) [inline .golangci.yml:18]: we could exclude the test files for this linter

Comment (vrothberg) [reply .golangci.yml:18]: Tests are just an example.  `dupword` seems to be checking any string and would also report things like `data := "0 0 0"` as a duplicate.  I don't think the linter is ready (for us).

**Review by Luap99 — COMMENTED** [review]
LGTM

File: .golangci.yml (hunk 1 above)
Comment (vrothberg) [reply .golangci.yml:18]: I don' think `dupword` is made for that use case.  It seems to only be looking for duplicates in strings and comments. "foo foo" for instance.

Comment (Luap99) [reply .golangci.yml:18]: oh you are right, I was thinking of `goconst`

**Comment by vrothberg** [comment]
@containers/podman-maintainers PTAL

**Pull request merged**